palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/stream/SerDesConfig.java

```diff
@@ -52,12 +52,12 @@ private SerDesConfig() {
      *
      * @return an appropriate value serialiser for the topic's message content
      */
-    public static Serializer<PalisadeRequest> requestSerializer() {
-        return (String ignored, PalisadeRequest palisadeRequest) -> {
+    public static Serializer<PalisadeSystemResponse> requestSerializer() {
+        return (String ignored, PalisadeSystemResponse request) -> {
```

Comment:
Why is this variable named request? If we're using the AuditableResponse class shouldn't this be named more appropriately?

Reply:
Named request because it goes onto the "request" topic on kafka